### PR TITLE
Raptor Changes:

### PR DESCRIPTION
**Pull request by EidolonForgery**
+ Gunner can now use Cargo interaction
+ Cargo space now set to 6
+ Inventory now has backup items
  - Rifles in inventory do not have attachments by default

Script Component Change:
+ Transport Items macros added

### CODE REVIEW

File: addons/vehicles/script_component.hpp [hunk 1]
```diff
@@ -35,4 +35,9 @@
 
 #define GREEN_NET_TEXTURE "A3\Armor_F\Data\camonet_nato_green_co.paa"
 #define CSAT_NET_TEXTURE "A3\Armor_F\Data\camonet_csat_hex_green_co.paa"
-#define AAF_CAGE_TEXTURE "A3\Armor_F\Data\cage_aaf_co.paa"
\ No newline at end of file
+#define AAF_CAGE_TEXTURE "A3\Armor_F\Data\cage_aaf_co.paa"
+
+/// Transport Items macros ///
+#define MAG_XX(var0,var1) class _xx_##var0 { magazine = QUOTE(var0); count = var1; }
```
Comment (WitchFreya) [inline addons/vehicles/script_component.hpp:41]: These already exist in addons\main\script_macros but you're welcome to rename those and refactor them elsewhere if you want to use this syntax instead

File: addons/vehicles/CfgVehicles/Raptor.inc [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
 class SWS_V_Heli_Raptor_Base : Heli_Attack_02_dynamicLoadout_base_F
-    {
+{
     ITEM_META(0);
     displayName = QNAME(Sx-48 Raptor - Base);
     faction = "SWS_UNSC";
     side = 1;
     crew = "B_UNSCAF_Pilot_01";
     typicalCargo[] = { "B_UNSCAF_Pilot_01" };
+    ace_cargo_space = 6;  //
```
Comment (WitchFreya) [inline addons/vehicles/CfgVehicles/Raptor.inc:11]: Looks like an errant comment slipped in.

**Review by WitchFreya — CHANGES_REQUESTED** [review]
mostly good, small tweaks

File: addons/vehicles/script_component.hpp (hunk 1 above)
Comment (EidolonForgery) [reply addons/vehicles/script_component.hpp:41]: Oh, I'll just use the base ones then. I didn't think to check that they were already defined, I should look at that file more often.

**Review by WitchFreya — APPROVED** [review]
lgtm, once the build annotations are done you can merge it

**Comment by WitchFreya** [comment]
in the squashed commit make the message something like
`feat(vehicles): Update Raptor cargo`

**Pull request merged**